articles/racket-lang.art

```diff
@@ -130,25 +130,25 @@
    their job.}
 
 @p{Luckily, there is a family of programming languages that have been
-   specifically designed to create DSLs. It's called @'|lisp|.}
+   specifically designed to create DSLs. It's called @'|Lisp|.}
 
-@p{In the lisp family, there are a lot of different languages. Some designed
+@p{In the Lisp family, there are a lot of different languages. Some designed
    for science, some designed for learning computer science, some general
    purposed, and some "language oriented".}
 
-@p{Racket is a language oriented language. It's my favorite lisp, and if you
+@p{Racket is a language oriented language. It's my favorite Lisp, and if you
    followed until now, you're now ready to discover some of the tricks that make
    this language incredibly powerful in creating domain specific languages.}
 
 )
 
-(section "A bit of lisp history"
+(section "A bit of Lisp history"
 
-@p{(You can skip this section if you want by clicking @link["#lists-and-atoms"]{here})}
+@p{(You can skip this section if you want by clicking @link["#lists-and-atoms"]{here}.)}
 
-@p{Racket is an implementation of Scheme, which is one of the two major lisp
+@p{Racket is an implementation of Scheme, which is one of the two major Lisp
    families (the other one being Common Lisp).
-   Both have been around for more than 60 years (you read that well) and mostly
+   Lisp has been around for more than 60 years (you read that well) and mostly
```

Comment:
I guess the sentence would make sense with "Lisps have been around [...] and mostly differ from each other" then.